requirements/edx/development.txt

```diff
@@ -229,14 +229,15 @@ pa11ycrawler==1.6.2
 packaging==17.1
 parsel==1.5.0
 path.py==8.2.1
+pathlib2==2.3.2
```

Comment:
Didn't this used to print which dependency it was getting pulled in for?  Did we turn that functionality off?

Reply:
That information is in `testing.txt`, the first file to pull it in.  The info isn't duplicated here because we used `-r testing.txt` as an input, so it's considered a direct requirement here.